nodeprobe/nodeprobe.go

```diff
@@ -2,164 +2,186 @@ package nodeprobe
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"sync"
-	"sync/atomic"
 	"time"
 
 	"go.uber.org/zap"
 )
 
+type nodeName string
+
 const (
-	probeInterval = 24 * time.Second
+	cl          = "consensus client"
+	el          = "execution client"
+	eventSyncer = "event-syncer"
 )
 
+// Node represents a node being probed.
 type Node interface {
 	Healthy(ctx context.Context) error
 }
 
+const (
+	probeFrequency = 60 * time.Second
+)
+
+// Prober probes(monitors) the nodes it is configured with. It can do it periodically in the background via
+// Start func (which can crash the process via logger.Fatal call if it finds out that one of the nodes being
+// probed is not healthy), or it can probe on demand via Probe func.
+// It is used to make sure the Ethereum nodes (CL, EL) are up and running, and they are healthy enough to for
```

Comment:
Since it may also probe the event syncer, we need to mention it along with the Ethereum nodes